### PR TITLE
Update Kubernetes docset to v1.7.5

### PR DESCRIPTION
**Pull request by ruoshan**
The doc content source is changed from kubernetes github pages to the
api reference included in kubernetes source code

### CODE REVIEW

**Comment by ruoshan** [comment]
@nextrevision are you okay with these changes?

**Comment by nextrevision** [comment]
LGTM! How manual was this process? I've tried to build some tooling around automatically generating docsets for K8S, but either their layout changes frequently, or there are too many corner cases. I'm curious about how you got it into this format, especially given their current API layout (https://kubernetes.io/docs/api-reference/v1.7/).

**Comment by ruoshan** [comment]
@nextrevision I fetch the documents from [here](https://github.com/kubernetes/kubernetes/tree/master/docs/api-reference)

**Comment by ruoshan** [comment]
@Kapeli Can you help me on this? I don't know how to fix the travis error. docset.json looks ok

File: docsets/Kubernetes/docset.json [hunk 1]
```diff
@@ -1,10 +1,20 @@
 {
     "name": "Kubernetes",
-    "version": "1.4",
+    "version": "1.7.5",
     "archive": "Kubernetes.tgz",
     "author": {
         "name": "John Patterson",
         "link": "https://twitter.com/cantrobot"
     },
     "aliases": ["kubernetes", "kubectl"]
```
Comment (Kapeli) [inline docsets/Kubernetes/docset.json]: You have a missing comma at the end of this line.

File: docsets/Kubernetes/docset.json [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+    "specific_versions": [
+        {
+            "version": "1.7.5",
+            "archive": "versions/1.7.5/Kubernetes.tgz"
+        },
+        {
+            "version": "1.4",
+            "archive": "versions/1.4/Kubernetes.tgz.txt"
```
Comment (Kapeli) [inline docsets/Kubernetes/docset.json]: This doesn't work. You need to add an actual `tgz` at this location.

Comment (ruoshan) [reply docsets/Kubernetes/docset.json]: I can't find the original archive anymore.

Comment (Kapeli) [reply docsets/Kubernetes/docset.json]: You can grab it from https://kapeli.com/feeds/zzz/user_contributed/build/Kubernetes/Kubernetes.tgz.

Comment (ruoshan) [reply docsets/Kubernetes/docset.json]: cool. fixed and rebased

Comment (Kapeli) [reply docsets/Kubernetes/docset.json]: The docset.json still needs to be updated to it points to the `tgz`, and not `tgz.txt`.

Comment (ruoshan) [reply docsets/Kubernetes/docset.json]: Fixed :(

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**